src/cleanvision/imagelab.py

```diff
@@ -289,15 +294,26 @@ def _visualize(
 
     def visualize(
         self,
-        issue_types: List[str],
+        filepaths: Optional[List[str]] = None,
```

Comment:
I recommend changing argument name `filepaths` -> `image_files` which I think is less ambiguous